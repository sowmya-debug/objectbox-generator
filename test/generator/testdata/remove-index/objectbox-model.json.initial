{
  "comment": [
    "KEEP THIS FILE! Check it into a version control system (VCS) like git.",
    "ObjectBox manages crucial IDs for your object model. See docs for details.",
    "If you have VCS merge conflicts, you must resolve them according to ObjectBox docs."
  ],
  "entities": [
    {
      "id": "1:8717895732742165505",
      "name": "A",
      "lastPropertyId": "2:6050128673802995827",
      "properties": [
        {
          "id": "1:2259404117704393152",
          "name": "Id"
        },
        {
          "id": "2:6050128673802995827",
          "name": "Removed",
          "indexId": "1:501233450539197794"
        }
      ]
    },
    {
      "id": "2:3390393562759376202",
      "name": "B",
      "lastPropertyId": "2:1774932891286980153",
      "properties": [
        {
          "id": "1:2669985732393126063",
          "name": "Id"
        },
        {
          "id": "2:1774932891286980153",
          "name": "Removed",
          "indexId": "2:6044372234677422456"
        }
      ]
    },
    {
      "id": "3:8274930044578894929",
      "name": "C",
      "lastPropertyId": "2:2661732831099943416",
      "properties": [
        {
          "id": "1:1543572285742637646",
          "name": "Id"
        }
      ]
    }
  ],
  "lastEntityId": "3:8274930044578894929",
  "lastIndexId": "3:8325060299420976708",
  "retiredEntityUids": [],
  "retiredIndexUids": [
    8325060299420976708
  ],
  "retiredPropertyUids": [
    2661732831099943416
  ],
  "version": 1
}