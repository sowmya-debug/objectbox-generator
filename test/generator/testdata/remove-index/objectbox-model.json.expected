{
  "_note1": "KEEP THIS FILE! Check it into a version control system (VCS) like git.",
  "_note2": "ObjectBox manages crucial IDs for your object model. See docs for details.",
  "_note3": "If you have VCS merge conflicts, you must resolve them according to ObjectBox docs.",
  "entities": [
    {
      "id": "1:8717895732742165505",
      "lastPropertyId": "2:6050128673802995827",
      "name": "A",
      "properties": [
        {
          "id": "1:2259404117704393152",
          "name": "Id",
          "type": 6,
          "flags": 1
        }
      ]
    },
    {
      "id": "2:3390393562759376202",
      "lastPropertyId": "3:7837839688282259259",
      "name": "B",
      "properties": [
        {
          "id": "1:2669985732393126063",
          "name": "Id",
          "type": 6,
          "flags": 1
        },
        {
          "id": "3:7837839688282259259",
          "name": "New",
          "indexId": "4:2518412263346885298",
          "type": 6,
          "flags": 8
        }
      ]
    },
    {
      "id": "3:8274930044578894929",
      "lastPropertyId": "3:5617773211005988520",
      "name": "C",
      "properties": [
        {
          "id": "1:1543572285742637646",
          "name": "Id",
          "type": 6,
          "flags": 1
        },
        {
          "id": "3:5617773211005988520",
          "name": "New",
          "indexId": "5:2339563716805116249",
          "type": 6,
          "flags": 8
        }
      ]
    }
  ],
  "lastEntityId": "3:8274930044578894929",
  "lastIndexId": "5:2339563716805116249",
  "lastRelationId": "",
  "modelVersion": 5,
  "modelVersionParserMinimum": 5,
  "retiredEntityUids": [],
  "retiredIndexUids": [
    8325060299420976708,
    501233450539197794,
    6044372234677422456
  ],
  "retiredPropertyUids": [
    2661732831099943416,
    6050128673802995827,
    1774932891286980153
  ],
  "retiredRelationUids": null,
  "version": 1
}